pkg/graph/resolvers/mto_solution.resolvers.go

```diff
@@ -0,0 +1,72 @@
+package resolvers
+
+// This file will be automatically regenerated based on the schema, any resolver implementations
+// will be copied through when generating and any unknown code will be moved to the end.
+// Code generated by github.com/99designs/gqlgen
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/cms-enterprise/mint-app/pkg/appcontext"
+	"github.com/cms-enterprise/mint-app/pkg/graph/generated"
+	"github.com/cms-enterprise/mint-app/pkg/models"
+	"github.com/google/uuid"
+)
+
+// RelatedMilestones is the resolver for the relatedMilestones field.
+func (r *mTOSolutionResolver) RelatedMilestones(ctx context.Context, obj *models.MTOSolution) ([]*models.MTOMilestone, error) {
+	panic(fmt.Errorf("not implemented: RelatedMilestones - relatedMilestones"))
+}
+
+// CommonSolution is the resolver for the commonSolution field.
+func (r *mTOSolutionResolver) CommonSolution(ctx context.Context, obj *models.MTOSolution) (*models.MTOCommonSolution, error) {
+	logger := appcontext.ZLogger(ctx)
+
+	return MTOSolutionGetCommonSolutionByKeyLoader(r.store, logger, obj.Key)
```

Comment:
This seems to be taking the wrong params -- I think we just need to pass `ctx` here too and it should be good!

```suggestion
	return MTOSolutionGetCommonSolutionByKeyLoader(ctx, r.store, logger, obj.Key)
```